manifests/config.pp

```diff
@@ -3,14 +3,21 @@
 # Controls the contents of falco.yaml and sets up log rotate, if needed
 #
 class falco::config inherits falco {
-  file { '/etc/falco/falco.yaml':
-    ensure  => file,
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0644',
-    content => template('falco/falco.yaml.erb'),
-    require => Class['falco::install'],
-    notify  => Service['falco'],
+  file {
+    default:
+      ensure  => file,
+      owner   => 'root',
+      group   => 'root',
+      mode    => '0644',
+      require => Class['falco::install'],
```

Comment:
These relationships are redundant as init.pp already declares that config comes after install and notifies service.

Reply:
I think I will leave this cleanup for a different PR since I also have not asserted these as private classes yet.